wrappers/react/src/helpers.tsx

```diff
@@ -159,13 +174,8 @@ export function createPortal(rElement: React.ReactElement, props, ownerDocument:
   const portalContainer = ownerDocument.createElement('DIV');
   bulkComponentContainer.appendChild(portalContainer);
 
-  const extendedRendererElement = React.cloneElement(rElement, {
-    key: `${props.row}-${props.col}`,
-    ...props
-  });
-
   return {
-    portal: ReactDOM.createPortal(extendedRendererElement, portalContainer, `${props.row}-${props.col}-${Math.random()}`),
```

Comment:
Good to see `Math.random()` removed, it was a funny one to find when I was fixing a re-render issue